apps/teams-test-app/src/components/SearchAPIs.tsx

```diff
@@ -0,0 +1,42 @@
+import { search } from '@microsoft/teams-js';
+import React, { ReactElement } from 'react';
+
+//import { search } from '../../../../packages/teams-js/src/public/search';
+import { ApiWithoutInput } from './utils';
+
+const RegisterHandlers = (): React.ReactElement =>
+  ApiWithoutInput({
+    name: 'search_registerHandlers',
+    title: 'Search Register Handlers',
+    onClick: async setResult => {
+      const onChange = (onChangeHandler: search.SearchQuery): void => {
+        console.log(onChangeHandler.searchTerm);
+
+        setResult('Update your application with the changed search query: ' + onChangeHandler.searchTerm);
+      };
+      const onClosed = (onClosedHandler: search.SearchQuery): void => {
+        console.log(onClosedHandler.searchTerm);
+        setResult(
+          'Update your application to handle the search experience being closed. Last query: ' +
+            onClosedHandler.searchTerm,
+        );
+      };
+      const onExecute = (onExecuteHandler: search.SearchQuery): void => {
+        console.log(onExecuteHandler.searchTerm);
+        setResult('Update your application to handle an executed search result: ' + onExecuteHandler.searchTerm);
+      };
+      setResult('register handlers');
+
+      search.registerHandlers(onChange, onClosed, onExecute);
+      return 'recieved';
+    },
+  });
+
+const SearchAPIs = (): ReactElement => (
+  <>
```

Comment:
This should use `<ModuleWrapper>` instead of manually creating the heading. See https://github.com/OfficeDev/microsoft-teams-library-js/pull/1263 for where it was added and how it's used in other places in the test app.